Sources/ReleaseServer/ReleaseServer.WebApi/JsonConverters/ProductVersionConverter.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReleaseServer.WebApi.Models;
+
+
+namespace ReleaseServer.WebApi.JsonConverters
+{
+    public class ProductVersionConverter : JsonConverter<ProductVersion>
+    {
+        public override void WriteJson(JsonWriter writer, ProductVersion value, JsonSerializer serializer)
+        {
+            writer.WriteValue(value.ToString());
+        }
+
+        public override ProductVersion ReadJson(JsonReader reader, Type objectType, ProductVersion existingValue, bool hasExistingValue,
+            JsonSerializer serializer)
+        {
+            return existingValue;
```

Comment:
The `ReadJson` function ins not needed in this project (only `ReleaseNotes` & `DeploymentMetaInfo` will be deserialized).

I replaced the implementation ith a `NotImplementedException`, because it's not needed and the former implementation does not work. The deseiralization is not so trivial as expected. The issues here: The `JsonReader` didn't parse the jsons as expected.

Changes see c3ae5d6

Reply:
Nevertheless, if you wish to implement this function, i can do it.
Actually i didn't want to waste too much time on a functionality that is not needed now ;)